### ISSUE TITLE
Input test data should have a valid name

### ISSUE THREAD

**Issue by snnn** [onnx/models#63]
How to reproduce:

Download squeezenet model from https://s3.amazonaws.com/download.onnx/models/opset_7/squeezenet.tar.gz
Unzip it and run the script below with squeezenet\test_data_set_0\input_0.pb

```python
import onnx
import sys

from onnx import TensorProto

proto = TensorProto()

# Read the existing address book.
f = open(sys.argv[1], "rb")
proto.ParseFromString(f.read())
f.close()
print(proto.name)
```

The output is empty.

We need this information to know this tensor is for which input.

**Comment by houseroad** [comment]
I will take a look and fix them.

**Comment by bddppq** [comment]
name in the input TensorProto is not required in onnx's spec

**Comment by snnn** [comment]
Then how could I know this pb file is for which input?

**Comment by snnn** [comment]
I tried to get such information from the model.graph.input, but right now it contains too many noises.  For example, the zfnet model, it's input contains the real input("gpu_0/data_0"), model weights, "gpu_0/imagenet1k_blobs_queue_e24a6638-b332-4e67-a127-91f5e17e2e11_0", "OC2_DUMMY_1". 

I think the simplest way is just fill that information in the TensorProtos.

**Comment by fs-eire** [comment]
+1 to the issue

**Comment by bddppq** [comment]
@snnn yeah I agree for the model zoo test data it would be better to have names in the TensorProto, just wanted to point out this is not in the spec and so backends shouldn't rely on it (if the backend happens to take TensorProto as inputs at runtime).

**Comment by fs-eire** [comment]
Is there update for the models in model zoo now?
